packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Placeholder events.  They will be fleshed out in their respective implementation issues.